pkg/helm/charts/tests/centralservices/flavor/flavor_test.go

```diff
@@ -52,30 +53,24 @@ func TestOverriddenTagsAreRenderedInTheChart(t *testing.T) {
 
 func TestWithDifferentImageFlavors(t *testing.T) {
 	testbuildinfo.SetForTest(t)
-	// having a function as value allows to successfully run this test without dependency to GOTAGS='' and GOTAGS='release'
-	imageFlavorCases := map[string]func() defaults.ImageFlavor{
-		"development": func() defaults.ImageFlavor {
-			testutils.SetVersion(t, testutils.GetExampleVersion(t))
-			return defaults.DevelopmentBuildImageFlavor()
-		},
-		"stackrox": func() defaults.ImageFlavor {
-			testutils.SetVersion(t, testutils.GetExampleVersionUnified(t))
-			return defaults.StackRoxIOReleaseImageFlavor()
-		},
-		"rhacs": func() defaults.ImageFlavor {
-			testutils.SetVersion(t, testutils.GetExampleVersionUnified(t))
-			return defaults.RHACSReleaseImageFlavor()
-		},
-		"custom": func() defaults.ImageFlavor {
-			return customFlavor(t)
-		},
+	testutils.SetVersion(t, testutils.GetExampleVersion(t))
+	imageFlavorCases := map[string]defaults.ImageFlavor{
+		"development": defaults.DevelopmentBuildImageFlavor(),
+		"stackrox":    defaults.StackRoxIOReleaseImageFlavor(),
```

Comment:
The reason we used callbacks is that `stackrox` and `rhacs` need to call `testutils.GetExampleVersionUnified(t)` rather than `testutils.GetExampleVersion(t)`. But I see that tests are passing and as long as the flavor is calling `version.GetAllVersionsDevelopment()` or `version.GetAllVersionsUnified()` we should be ok. I'm just puzzled why this was there in the first place. Maybe we don't need it after all.

Reply:
Flavors indeed call `version.GetAllVersionsDevelopment()` or `version.GetAllVersionsUnified()` and the latter one does this:
https://github.com/stackrox/stackrox/blob/c9392345af59a237ae52ec4b56fee9bfed156d99/pkg/version/version.go#L66-L71

Therefore we don't need to use `testutils.GetExampleVersionUnified(t)`.
I suppose the old setup was a result of either over-editing or back then things were working differently.